configuration.yaml

```diff
@@ -84,3 +85,7 @@ Encore:
   data_directory: gs://otar013-ppp/encore/input
   config: gs://otar013-ppp/PPP-evidence-configuration/encore_config.json
   outputBucket: gs://otar013-ppp/encore
+CrisprScreens:
+  outputBucket: gs://otar000-evidence_input/Crispr_screens/json
+  # This line needs to be updated once the mapping is merged into the curation repo:
+  crispr_brain_mapping: https://gist.githubusercontent.com/DSuveges/218e1a5747b287423654ec16a0b37124/raw/2becccda79c9b7a42a139212e4ac3c247cf6611a/brain_crispr_studies.tsv
```

Comment:
Is there something blocking us from merging this into the curation repo straight away? I'd really prefer not to merge a hardcoded gist link into the main branch here :sweat_smile: